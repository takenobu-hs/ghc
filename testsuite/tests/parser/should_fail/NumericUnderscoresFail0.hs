{-# LANGUAGE NumericUnderscores #-}

main :: IO ()
main = do
    -- invalid case for NumericUnderscores extension
    print [
            -- integer
            1000000_,
            _1000000,

            -- float
            0_.0001,
            _0.0001,
            0.0001_,
            0._0001,

            -- float with exponent
            1e_+23,
            1e+23_,
            1e+_23
          ]
