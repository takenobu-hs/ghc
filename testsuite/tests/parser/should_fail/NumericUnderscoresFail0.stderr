NumericUnderscoresFail0.hs:9:5: error:
NumericUnderscoresFail0.hs:11:13: error:
NumericUnderscoresFail0.hs:11:20: error:
NumericUnderscoresFail0.hs:12:13: error:
